published/088-2020-may-5.md

```diff
@@ -0,0 +1,86 @@
+# Welcome to the IPFS Weekly
+
+Here’s what’s happening lately in the [InterPlanetary File System](https://ipfs.io/) galaxy and beyond! 🚀
+
+## Top story: 
+* 
+
+
+## New bounties!
+https://github.com/ipfs/devgrants/projects/1
+
+
+## Using IPFS to fight COVID-19
+
++ Origin Protocol Launches Fundraiser to Help Find Vaccine for COVID-19 https://www.coinspeaker.com/origin-protocol-launches-fundraiser-to-help-find-vaccine-for-covid-19/
++ Privacy-first COVID-19 app TRACY chooses Matic Network https://medium.com/moibit/privacy-first-covid-19-app-tracy-chooses-matic-network-633f1bba4c66
++ 
+
+## Latest releases
+* [Release v0.11.1 · ipfs-shipyard/ipfs-desktop · GitHub](https://github.com/ipfs-shipyard/ipfs-desktop/releases/tag/v0.11.1)
+* [Alex Potsides on Twitter: “js-IPFS will be getting the same improvements to bitswap (e.g. transfer speeds) very soon!   Take a look at the roadmap here: https://t.co/OAhLUkKx5P… https://t.co/mEbkhnfMaJ”](https://twitter.com/achingbrain/status/1255562111436103680?s=20)
+* https://medium.com/equilibriumco/rust-ipfs-append-only-log-001-42106983f3c
+
+## Video
+https://www.youtube.com/watch?v=G8FvB_0HlCE&feature=emb_logo
+
+## 0.5 Release news
+* [Decentralized Web Protocol IPFS Has Its Biggest Update So Far](https://cointelegraph.com/news/decentralized-web-protocol-ipfs-has-its-biggest-update-so-far)
+* [The InterPlanetary File System gets a network upgrade to make data fetching faster - The Block](https://www.theblockcrypto.com/post/63189/ipfs-network-upgrade-data-fetching)
+* [Latest Version of Open Source IPFS Improves Performance - DevOps.com](https://devops.com/latest-version-of-open-source-ipfs-improves-performance/)
+* [IPFS Releases 0.5.0 Network Upgrade, Finding and Fetching Data Is Now 5 Times Faster](https://bitcoinexchangeguide.com/ipfs-releases-0-5-0-network-upgrade-finding-and-fetching-data-is-now-5-times-faster/)
+* [Ya fue liberada la nueva versión del sistema de archivos descentralizado IPFS 0.5](https://blog.desdelinux.net/ya-fue-liberada-la-nueva-version-del-sistema-de-archivos-descentralizado-ipfs-0-5/)
+* [Protocolo IPFS é atualizado para tornar coleta de dados mais rápida – Money Times](https://www.moneytimes.com.br/protocolo-ipfs-e-atualizado-para-tornar-coleta-de-dados-mais-rapida/)
+
+
+## Top highlights this week
+* [IPFS and the Future of the Internet - DLT Labs - Medium](https://medium.com/@dltlabs/ipfs-and-the-future-of-the-internet-dfb3cfed0531)
+* 3 Part Tutorial: [Nim-libp2p Tutorial: A Peer-to-Peer Chat Example (1)](https://our.status.im/nim-libp2p-tutorial-a-peer-to-peer-chat-example-1/)
+* [Algorand Developer Portal](https://developer.algorand.org/solutions/creating-point-sale-application-algorand-blockchain/)
```

Comment:
Please replace text "Algorand Developer Portal" with "Creating a Point-of-Sale System with the Algorand Blockchain and IPFS".